controllers/clusters/postgresql_controller.go

```diff
@@ -282,6 +282,49 @@ func (r *PostgreSQLReconciler) handleCreateCluster(
 	}
 
 	if pg.Spec.UserRefs != nil {
+		nodeList := &k8sCore.NodeList{}
+
+		err = r.List(ctx, nodeList, &client.ListOptions{})
+		if err != nil {
+			logger.Error(err, "Cannot list all nodes",
+				"cluster name", pg.Spec.Name,
+				"clusterID", pg.Status.ID,
+			)
+
+			r.EventRecorder.Eventf(
+				pg, models.Warning, models.CreationFailed,
+				"Fetching all nodes is failed. Reason: %v",
+				err,
+			)
+
+			return models.ReconcileRequeue
+		}
+
+		externalIPExists := false
```

Comment:
externalIPExists := false -> var externalIPExists bool  default value will be false wdyt?

Reply:
Yea, both are good, changed